app/models/alert.rb

```diff
@@ -7,13 +7,14 @@
 #  analytics_valid         :boolean          default(TRUE), not null
 #  chart_data              :json
 #  created_at              :datetime         not null
+#  custom_period_id        :bigint(8)
 #  displayable             :boolean          default(TRUE), not null
 #  enough_data             :integer
 #  id                      :bigint(8)        not null, primary key
 #  priority_data           :json
 #  rating                  :decimal(, )
 #  relevance               :integer          default("relevant")
-#  report_period           :integer
+#  reporting_period        :integer
```

Comment:
is reporting_period needed if custom_period_id is there to indicate a custom period?

Reply:
The field is there not just to indicate if there's a custom period. There are other reporting periods we just haven't defined or started populating them yet. E.g. "last 2 school weeks", "last 2 holidays", "last school year". These are all relative periods so can't be statically defined. There's a card "Add Additional Reporting Periods" in the backlog.